scss/_navbar-offcanvas.scss

```
@@ -3,6 +3,7 @@
   display: none;
 }
 .offcanvas-toggle-body-freeze {
+  position: fixed;
   overflow-y: hidden;
```

Looks like upstream Bootstrap 4.x just sets `overflow: hidden` and does not set position fixed which also seems to work with Safari 🤷🏻‍♂️.
```suggestion
  overflow: hidden;
```